src/style.css

```diff
@@ -1 +1,123 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  padding: 0;
+  font-size: 16px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
```

Comment:
Don't fix container sizes unless required. Here, the `.search-form` has a fixed margin, which might not be necessary. Let the content size dictate the container size to avoid overflow or accidental scroll bars.